SwiftUIHW/SwiftUIHW/ContentView.swift

```diff
@@ -7,21 +7,177 @@
+        }
+        
+            createCell(imageName: Constants.wifiImageName, text: Constants.wifiText, color: .blue, rightText: Constants.beelineText, type: .simple)
+        
+        Picker(selection: $section) {
+            let count = blueStatus.count
+            ForEach(Constants.zero..<count) { index in
+                Text(blueStatus[index])
+            }
+        } label: {
+            createCell(imageName: Constants.bluetoothImageName, text: Constants.bluetoothText, color: .blue, rightText: nil, type: .modify)
+        }
+            createCell(imageName: Constants.antennaImageName, text: Constants.mobileConnectionText, color: .green, rightText: nil, type: .simple)
+            createCell(imageName: Constants.modemImageName, text: Constants.modemText, color: .green, rightText: nil, type: .simple)
+            createCell(imageName: Constants.vpnImageName, text: Constants.vpnText, color: .blue, rightText: blueStatus.first, type: .simple)
+    }
+}
+    
+    private var createFourSection: some View {
+        return Section {
+            createCell(imageName: Constants.bellImageName, text: Constants.notifyText, color: .red, rightText: nil, type: .simple)
+            createCell(imageName: Constants.speakerImageName, text: Constants.volumeText, color: .red, rightText: nil, type: .simple)
+            createCell(imageName: Constants.moonImageName, text: Constants.focusText, color: .specialPurple, rightText: nil, type: .simple)
+            createCell(imageName: Constants.clockImageName, text: Constants.screenTimeText, color: .specialPurple, rightText: nil, type: .simple)
+        }
+    }
+    
+    fileprivate func createCell(imageName: String, text: String, color: Color, rightText: String?, type: CellTypes) -> some View {
+        return HStack(spacing: Constants.cellHstackSpacing) {
+            Image(systemName: imageName).padding().frame(width: Constants.iconFrameValue, height: Constants.iconFrameValue)
+                .foregroundColor(Color.white).background(RoundedRectangle(cornerRadius: Constants.iconCornerRadius)).foregroundColor(color)
+            Text(text)
+            if type == .simple {
+                Spacer()
+                Text(rightText ?? Constants.emptyString).foregroundColor(.gray)
+                Image(systemName: Constants.chevronImageName).foregroundColor(.gray)
+            }
+        }
+    }
+    
+    fileprivate func notifyCircle(_ number: String) -> some View {
+        return ZStack {
+            Circle().frame(width: Constants.circleWidthHeightValue, height: Constants.circleWidthHeightValue).foregroundColor(.red)
+            Text(number).foregroundColor(.white)
         }
-        .padding()
     }
 }
 
-
+/// Превью контента
 struct ContentView_Previews: PreviewProvider {
     static var previews: some View {
         ContentView()
     }
 }
+
+/// Constants and CellTypes
```

Comment:
```suggestion
/// Constants and CellTypes
```
в других местах описание на русском